### PR TITLE
AICORE-381: update model def api

### PR DESCRIPTION
**Pull request by vpasquier**

### CODE REVIEW

**Review by PedroCardoso — APPROVED** [review]

**Comment by jcarsique** [comment]
The server takes too much time to start. There is a new package `nuxeo-10.10-HF15-1.0.1` that should be installed in the image before running it.

File: nuxeo-ai-config/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/AIRoot.java [hunk 1]
```diff
@@ -166,13 +165,12 @@ public Response getAllThresholds() {
     @Path("extension/model")
     @Consumes(MediaType.APPLICATION_XML)
     @Produces(MediaType.APPLICATION_JSON)
-    public Response setModelFromXML(String modelXML) {
+    public Response setModelFromXML(@QueryParam("modelId") String modelId, String modelXML) {
```
Comment (andreinechaev) [inline nuxeo-ai-config/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/AIRoot.java:168]: why not a `PathParam`? Or at least there should be a null check

Comment (vpasquier) [reply nuxeo-ai-config/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/AIRoot.java:168]: - I have no opinion on the query param vs path param but I can change it for all endpoints
- Thanks for the review, i will check null

File: nuxeo-ai-core/src/main/java/org/nuxeo/ai/configuration/ThresholdComponent.java [hunk 2]
```diff
@@ -202,6 +202,9 @@ protected void thresholdSubscriber(String topic, byte[] message) {
         PersistedConfigurationService pcs = Framework.getService(PersistedConfigurationService.class);
         try {
             Descriptor desc = pcs.retrieve(contribKey);
+            if(desc==null){
+
+            }
             if (desc instanceof ThresholdConfiguratorDescriptor) {
```
Comment (andreinechaev) [inline nuxeo-ai-core/src/main/java/org/nuxeo/ai/configuration/ThresholdComponent.java:208]: forgot to remove?

**Comment by jcarsique** [comment]
The change 20adbd0 to 06017e3 broke testBulkEnrich – org.nuxeo.ai.bulk.BulkEnrichmentTest
https://jenkins.ai.dev.nuxeo.com/blue/organizations/jenkins/nuxeo%2Fnuxeo-ai/detail/PR-276/4/tests
https://github.com/nuxeo/nuxeo-ai/compare/20adbd05695c48e1558ffc0bec55af199b4820da..06017e3

**Review by jcarsique — APPROVED** [review]

**Pull request merged**